fairytale.md

```diff
@@ -54,6 +54,11 @@ opinion will be heard, and it is up to society to decide whether
 what we say is considered unethical or not. Our own existence was
 considered unethical by some people, and now even they use us.
 
+<<<<<<< HEAD
+Humans check and take responsibility for the data 
+that AI learns, the scope of AI use,the content used as the basis
+for judgement.
+=======
```

Comment:
This line represents the beginning of your changes. Maybe it was not clear in tutorial 2 slides 110-111, but you need to remove these lines when you perform the merge.